tests/extra_setuptools/test_setuphelper.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import subprocess
+from textwrap import dedent
+
+DIR = os.path.abspath(os.path.dirname(__file__))
+MAIN_DIR = os.path.dirname(os.path.dirname(DIR))
+
+
+def test_simple_setup_py(monkeypatch, tmpdir):
+    monkeypatch.chdir(tmpdir)
+    monkeypatch.syspath_prepend(MAIN_DIR)
+
+    (tmpdir / "setup.py").write_text(
+        dedent(
+            u"""\
+        import sys
```

Comment:
Something weird happened to the indentation here?

Reply:
Probably my fault, Black won't change a string internally.